documentation/assemblies/metrics/assembly-metrics.adoc

```diff
@@ -6,34 +6,30 @@
 = Introducing metrics
 
 [role="_abstract"]
-You can use Prometheus and Grafana to monitor your Strimzi deployment.
+Collecting metrics is critical for understanding the health and performance of your Kafka deployment. 
+By monitoring metrics, you can actively identify issues before they become critical and make informed decisions about resource allocation and capacity planning. Without metrics, you may be left with limited visibility into the behavior of your Kafka deployment, which can make troubleshooting more difficult and time-consuming. Setting up metrics can save you time and resources in the long run, and help ensure the reliability of your Kafka deployment.
 
-You can monitor your Strimzi deployment by viewing key metrics on dashboards and setting up alerts that trigger under certain conditions.
-Metrics are available for each of the components of Strimzi.
+Metrics are available for each Strimzi component, allowing you to gain insights into their individual performance. 
+Additionally, you can collect metrics specific to `oauth` authentication and `opa` or `keycloak` authorization by enabling the `enableMetrics` property in the listener configuration of the `Kafka` resource. 
+Similarly, you can enable metrics for `oauth` authentication in custom resources such as `KafkaBridge`, `KafkaConnect`, `KafkaMirrorMaker`, and `KafkaMirrorMaker2`.
 
-You can also collect metrics specific to `oauth` authentication and `opa` or `keycloak` authorization. 
-You do this by setting the `enableMetrics` property to `true` in the listener configuration of the `Kafka` resource. 
-For example, set `enableMetrics`  to `true` in `spec.kafka.listeners.authentication` and `spec.kafka.authorization`.
-Similarly, you can enable metrics for `oauth` authentication in the `KafkaBridge`, `KafkaConnect`, `KafkaMirrorMaker`, and `KafkaMirrorMaker2` custom resources.  
+You can use Prometheus and Grafana to monitor Strimzi.
+Prometheus consumes key metrics from the running pods in your cluster when configured with Prometheus rules. 
+Grafana visualizes these metrics on dashboards, providing an intuitive interface for monitoring. 
 
-To provide metrics information, Strimzi uses Prometheus rules and Grafana dashboards.
+To facilitate metrics integration, Strimzi provides example Prometheus rules and Grafana dashboards for Strimzi components. 
+You can customize the example Grafana dashboards to suit your specific deployment requirements.
+You can use rules to define conditions that trigger notifications based on specific metrics.
```

Comment:
Let's use "alerts" instead of "notifications" as per Prometheus naming (i.e. alert manager)